### ISSUE TITLE
Get email working on live erver and move email config into .env files (and .config.js)

### ISSUE THREAD

**Issue by utunga** [CredComSoc/CreditCoopTech#84]
This info should not be / cannot be hard coded into the app it needs to be in .env files and passwords in .env,production.local file which is *not* checked in to github. 

<img width="489" alt="image" src="https://github.com/CredComSoc/CreditCoopTech/assets/166867/12647b5a-9625-45c2-ae31-2b5e5f1380d5">

This is vital, because this repo is open source and public. Also we need to change these passwords once we have made this change.

**Comment by utunga** [comment]
nodemailer may need a different configuration method entirely actually so need to look into that

**Comment by utunga** [comment]
@yonasbek I made a start on moving things into .env file in the email_sender_configuration  branch.

I'm gonna leave the rest of this card to you though - to configure nodemailer (or perhaps replace it completely).

The code I did on this branch is completely untested but hopefully of some use. Feel free to ignore it if you want to do it from scratch - just so long as we don't have passwords in code we should be good.

https://github.com/CredComSoc/CreditCoopTech/tree/email_sender_configuration
